server/handleSQS.js

```diff
@@ -0,0 +1,65 @@
+import {} from 'dotenv/config';
+import { Promise } from 'bluebird';
+import Consumer from 'sqs-consumer';
+import sql from '../database/index';
+
+
+// Set the long-polling tools on incoming queues
+
+const queueFromReservations = Consumer.create({
+  queueUrl: process.env.SQS_QUEUE_URL,
+  handleMessage: (message, done) => {
+    console.log('Message from queue1: ', message);
+    handleResMess(message) // eslint-disable-line no-use-before-define
+      .then(() => done());
+  }
+});
+queueFromReservations.on('sqs-consumer error: ', err => console.log(err.message));
+queueFromReservations.start();
+
+/*
+Message from reservations will look like this:
+  { dateAvailability: { '3': [1, 2, 1, 12, 3, 5, 5, 7] },
+    rental: '9bd9dac7-8020-4dbd-890d28' }
+*/
+function handleResMess(message) {
+  const dates = [];
+  Object.keys(message.dateAvailability).forEach(el => {
```

Comment:
I will most likely be sending you the entire month's array (from index 0 to 30) all with updated availabilities, will that be okay? Because it seems like you're updating the availabilities on your end as well.

Reply:
What would the updated availabilities look like? Are they just not included in the date array or have null values?